### PR TITLE
added the view orders page and its components

### PR DESCRIPTION
**Pull request by ShivaKakarla512**
WIP

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1151,6 +1151,29 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
+"@coreui/coreui@^4.0.1":
```
Comment (dhee9000) [inline yarn.lock:1154]: same as package.json

File: src/views/viewOrdersStatus/components/ViewOrder.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	CBadge,
+	CProgress,
+	CProgressBar,
+	CDropdown,
+	CDropdownToggle,
+	CDropdownMenu,
+	CDropdownItem,
+} from '@coreui/react';
+
+type Props = {
```
Comment (dhee9000) [inline src/views/viewOrdersStatus/components/ViewOrder.tsx:18]: Similarly, IViewOrderProps

File: src/views/viewOrdersStatus/components/ViewOrder.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	value: string;
+	tootlTip: string;
+	status: string;
+	statusColor: string;
+	progress: number;
+	subtitle: string;
+};
+
+type State = {
```
Comment (dhee9000) [inline src/views/viewOrdersStatus/components/ViewOrder.tsx:27]: Similarly, IViewOrderState

**Review by dhee9000 — CHANGES_REQUESTED** [review]
Also looks great, I left some feedback, once it's addressed this one is good to approve too.

**Review by koletimanish — APPROVED** [review]
Looks good to me!

**Review by dhee9000 — APPROVED** [review]
Looks great! Ready to merge in to develop

**Review by dhee9000 — APPROVED** [review]
Looks good!

**Pull request merged**